Project-Aurora/Project-Aurora/Devices/Omen/OmenFourZoneLighting.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Drawing;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Newtonsoft.Json.Linq;
+using System.Management;
+
+using Aurora.Settings;
+
+namespace Aurora.Devices.Omen
+{
+    class OmenFourZoneLighting : IOmenDevice
```

Comment:
I don't know how to feel about this whole class. Honestly the WMI stuff feels like maybe it belongs in the sdk dll and not in aurora? It also has a lot of magic numbers that dont properly describe what is being done.

Besides that, Aurora doesn't really have a way to properly support zoned devices, but the way it's done here is a bit strange. It might be worth adding DeviceKeys entries for the zones, but they won't have default profiles most of the time.

Reply:
Hi @diogotr7, 

How do you think four zone lighting denpends the keys, ADDITIONALLIGHT1 - 4 ? We treat them as four additional light.